ietf/doc/templatetags/ietf_filters.py

```diff
@@ -142,45 +125,11 @@ def bracketpos(pos,posslug):
 
 register.filter('fill', fill)
 
-@register.filter(name='rfcspace')
-def rfcspace(string):
-    """
-    If the string is an RFC designation, and doesn't have
-    a space between 'RFC' and the rfc-number, a space is
-    added
-    """
-    string = str(string)
-    if string[:3].lower() == "rfc" and string[3] != " ":
-        return string[:3].upper() + " " + string[3:]
-    else:
-        return string
-
-@register.filter(name='rfcnospace')
-def rfcnospace(string):
-    """
-    If the string is an RFC designation, and does have
-    a space between 'RFC' and the rfc-number, remove it.
-    """
-    string = str(string)
-    if string[:3].lower() == "rfc" and string[3] == " ":
-        return string[:3] + string[4:]
-    else:
-        return string
-
 @register.filter
 def prettystdname(string):
     from ietf.doc.utils import prettify_std_name
```

Comment:
This function takes an argument named `spacing`, and the filter could be modified to take the argument as well. In the places this filter is used to replace `rfcnospace`, the filter could then be called with `prettystdname:""`